iati/about/templates/about/people_page.html

```diff
@@ -0,0 +1,72 @@
+{% extends "base.html" %}
```

Comment:
Request changes:

The CSS doesn't seem to be showing correctly when replicating part of https://iati.surge.sh/page-templates/01-4-people (particularly in the `subheading` field).

Also a default image is needed when the user does not add a profile photo.

![image](https://user-images.githubusercontent.com/8247168/39056128-bb08cd38-44ad-11e8-9107-bd121bbb2906.png)

Reply:
Also, the template needs a little refactoring in order to accomodate @akmiller01 's suggested changes to the model - see above.